dali/pipeline/operators/fused/resize_crop_mirror.h

```diff
@@ -72,14 +72,33 @@ class ResizeCropMirrorAttr : protected CropAttr {
     if (resize_shorter_) {
       // resize_shorter set
       const int shorter_side_size = spec.GetArgument<float>("resize_shorter", ws, index);
+
+      bool max_size_enforced = spec.ArgumentDefined("max_size");
```

Comment:
Can we set it only once in the constructor. Now we will call it for each sample in the setup params.

Reply:
I think that we need it here because I  set `enable_tensor_input` to true.
If it might be really useful is an open question.